rel-eng/atomic-enterprise-node.service

```diff
@@ -11,7 +9,7 @@ Documentation=https://github.com/openshift/origin
 Type=notify
 EnvironmentFile=/etc/sysconfig/atomic-enterprise-node
 Environment=GOTRACEBACK=crash
-ExecStart=/usr/bin/atomic-enterprise start node --config=${CONFIG_FILE} $OPTIONS
+ExecStart=/usr/bin/openshift start node --config=${CONFIG_FILE} $OPTIONS
```

Comment:
For now it should be ```atomic-enterprise```. Future work will move this to configuration rather than binary check AFAIK.